libs/blocks/carousel/carousel.js

```diff
@@ -496,6 +496,34 @@ export default function init(el) {
   nextPreviousContainer.append(...nextPreviousBtns, controlsContainer);
   el.append(nextPreviousContainer);
 
+  function normalizeVideoHeights() {
```

Comment:
If the window is resized, the video parents might also get resized, thus the normalized video height might no longer apply. Should this be run on resize (debounced) as well?